docs/checks/deprecate_lazysizes.md

```diff
@@ -0,0 +1,61 @@
+# Deprecate lazySizes (`DeprecateLazysizes`)
+
+lazySizes is a common Javascript library used to lazy load images, iframes and scripts. 
```

Comment:
```suggestion
[lazysizes](https://github.com/aFarkas/lazysizes) is a common JavaScript library used to lazy load images, iframes and scripts.
```

Makes for easier browsing if you don't know what it is. (Also it looks like it's spelled with all lower case)

And JavaScript takes a capital S.

Sorry being super nitpicky :upside_down_smile: